### ISSUE TITLE
Некорректный тип функции `it(`

### ISSUE THREAD

**Issue by dicrtarasov** [gemini-testing/hermione#847]
### Last Hermione version that worked

8.0.5

### Hermione version

8.1.0

### Code to reproduce the issue

```typescript
it("Мой тест", async ({currentTest}) => {
    ....
});
```

В новой версии ругается что `currentTest` не определена в `Mocha.TestFunction`, хотя ранее тут был тип `TestDefinition` от Hermione.

### Expected behavior

syntax correct

### Actual behavior

TS2339: Property 'currentTest' does not exist on type 'Done'.

### Node.js version

20.10.0

### Operating System

macOS

**Comment by dicrtarasov** [comment]
This does not helps :((

![image](https://github.com/gemini-testing/hermione/assets/35159934/6eb67ace-bbfb-49f5-ae01-362a66c5825f)

![image](https://github.com/gemini-testing/hermione/assets/35159934/04174e19-dac5-4bc9-996a-6c0e55da9bb4)

But this is workaround:

![image](https://github.com/gemini-testing/hermione/assets/35159934/9a32d0b1-e526-40ca-9b30-cab9fb703597)

**Comment by shadowusr** [comment]
You are right. Only your workaround or similar triple-slash directive works.

I'm working on this issue now and will let you know as soon as I figure out a solution.

**Comment by shadowusr** [comment]
After discussion with the team, I believe the only viable solution for us would be to publish typings separately in the `@types` package, even though hermione itself is now written in TypeScript. This way, everything would work as expected and global API would be visible automatically. Jest uses the same approach. I tried to do this quickly, but encountered some deeper types issues in one of our dependencies, so for now I'll put it into backlog.

Up until that, I suggest using the workaround you mentioned. We'll get back to this issue as soon as we can.
